### PR TITLE
Rework Main Section of Actor Sheet [Reopen PR]

### PR DESCRIPTION
**Pull request by moliloo**
Styling Main Section for a better of the sheet and a better use, the gap between the columns is intentional because of the width of the sheet, in future merges, reworking other partials, i want, in the future, to reduce the width of the Actor Sheet.

### CODE REVIEW

File: lang/en.json [hunk 1]
```diff
@@ -375,6 +375,7 @@
             "DerStat.HP": "Max HP",
             "DerStat.Shield": "Shield",
             "DerStat.Sta": "STA",
+            "DerStat.Toxicity": "Toxicity",
```
Comment (Stexinator) [inline lang/en.json:378]: Can you move this to stats?

File: module/actor/sheets/WitcherActorSheet.js [hunk 2]
```diff
@@ -1,240 +1,238 @@
...
+import { criticalWoundMixin } from "./mixins/criticalWoundMixin.js"
+import { noteMixin } from "./mixins/noteMixin.js"
+import { globalModifierMixin } from "./mixins/globalModifierMixin.js"
+import { skillModifierMixin } from "./mixins/skillModifierMixin.js"
+import { skillMixin } from "./mixins/skillMixin.js"
+import { statMixin } from "./mixins/statMixin.js"
+import { itemMixin } from "./mixins/itemMixin.js"
 
-import { itemContextMenu } from "./interactions/itemContextMenu.js";
+import { itemContextMenu } from "./interactions/itemContextMenu.js"
```
Comment (Stexinator) [inline module/actor/sheets/WitcherActorSheet.js:15]: You are removing ;

File: module/actor/sheets/WitcherActorSheet.js [hunk 3]
```diff
@@ -1,240 +1,238 @@
...
-    return total
+	var total = 0
+	for (var i = 0; i < this.length; i++) {
+		if (this[i].system[prop]) {
+			total += Number(this[i].system[prop])
+		} else if (this[i].system?.system[prop]) {
+			total += Number(this[i].system?.system[prop])
+		}
+	}
+	return total
```
Comment (Stexinator) [inline module/actor/sheets/WitcherActorSheet.js:26]: whitespaces changed

**Review by Stexinator — CHANGES_REQUESTED** [review]

File: lang/en.json (hunk 1 above)
Comment (moliloo) [reply lang/en.json:378]: I'll do this

File: module/actor/sheets/WitcherActorSheet.js (hunk 2 above)
Comment (moliloo) [reply module/actor/sheets/WitcherActorSheet.js:15]: Hmm, maybe this ia my editor remove ; i will fix that too

File: module/actor/witcherActor.js [hunk 4]
```diff
@@ -29,52 +29,54 @@ export default class WitcherActor extends Actor {
         const baseMax = Math.floor((stats.body.max + stats.will.max) / 2);
         const meleeBonus = Math.ceil((stats.body.current - 6) / 2) * 2;
 
-        let intTotalModifiers = this.getAllModifiers("int").totalModifiers;
```
Comment (Stexinator) [inline module/actor/witcherActor.js:32]: Here is still an issue

File: module/actor/witcherActor.js [hunk 5]
```diff
@@ -220,7 +241,8 @@ export default class WitcherActor extends Actor {
             if (attribute.name == "emp" && (skillName == "charisma" || skillName == "leadership" || skillName == "persuasion" || skillName == "seduction")) {
                 if (tolerated.includes(this.system.general.socialStanding)) {
                     rollFormula += !displayRollDetails ? `-1` : `-1[${game.i18n.localize("WITCHER.socialStanding.tolerated")}]`;
-                } else if (hated.includes(this.system.general.socialStanding)) {
+                }
+					 else if (hated.includes(this.system.general.socialStanding)) {
```
Comment (Stexinator) [inline module/actor/witcherActor.js:245]: what happened here?

**Review by Stexinator — CHANGES_REQUESTED** [review]

**Review by Stexinator — APPROVED** [review]

**Comment by Stexinator** [comment]
Can you fix the conflict and squash the commits? Then I will merge it. Thanks

**Pull request closed without merging**